### ISSUE TITLE
Fails with Webpack 4

### ISSUE THREAD

**Issue by fi5u** [scinos/webpack-plugin-hash-output#13]
When used with Webpack 4, the following error is emitted:

```
 69% building modules 1345/1350 modules 5 active ...styles/vendor/bootstrap-dropdown.scss/Users/fisu/Projects/yoo/node_modules/webpack/lib/Chunk.js:438
		throw new Error("Chunk.parents: Use ChunkGroup.getParents() instead");
```

With the following traceback:
```
Error: Chunk.parents: Use ChunkGroup.getParents() instead
    at Chunk.get (/Users/fisu/Projects/yoo/node_modules/webpack/lib/Chunk.js:438:9)
    at compilation.plugin (/Users/fisu/Projects/yoo/node_modules/webpack-plugin-hash-output/src/OutputHash.js:140:34)
    at SyncHook.eval [as call] (eval at create (/Users/fisu/Projects/yoo/node_modules/tapable/lib/HookCodeFactory.js:17:12), <anonymous>:7:1)
    at SyncHook.lazyCompileHook [as _call] (/Users/fisu/Projects/yoo/node_modules/tapable/lib/Hook.js:35:21)
    at hooks.optimizeAssets.callAsync.err (/Users/fisu/Projects/yoo/node_modules/webpack/lib/Compilation.js:952:38)
    at AsyncSeriesHook.eval [as callAsync] (eval at create (/Users/fisu/Projects/yoo/node_modules/tapable/lib/HookCodeFactory.js:24:12), <anonymous>:6:1)
    at AsyncSeriesHook.lazyCompileHook [as _callAsync] (/Users/fisu/Projects/yoo/node_modules/tapable/lib/Hook.js:35:21)
    at hooks.optimizeChunkAssets.callAsync.err (/Users/fisu/Projects/yoo/node_modules/webpack/lib/Compilation.js:948:32)
    at _err0 (eval at create (/Users/fisu/Projects/yoo/node_modules/tapable/lib/HookCodeFactory.js:24:12), <anonymous>:11:1)
    at /Users/fisu/Projects/yoo/node_modules/uglifyjs-webpack-plugin/dist/index.js:262:11
    at _class.runTasks (/Users/fisu/Projects/yoo/node_modules/uglifyjs-webpack-plugin/dist/uglify/index.js:63:9)
    at UglifyJsPlugin.optimizeFn (/Users/fisu/Projects/yoo/node_modules/uglifyjs-webpack-plugin/dist/index.js:182:16)
    at AsyncSeriesHook.eval [as callAsync] (eval at create (/Users/fisu/Projects/yoo/node_modules/tapable/lib/HookCodeFactory.js:24:12), <anonymous>:7:1)
    at AsyncSeriesHook.lazyCompileHook [as _callAsync] (/Users/fisu/Projects/yoo/node_modules/tapable/lib/Hook.js:35:21)
    at hooks.additionalAssets.callAsync.err (/Users/fisu/Projects/yoo/node_modules/webpack/lib/Compilation.js:943:36)
    at AsyncSeriesHook.eval [as callAsync] (eval at create (/Users/fisu/Projects/yoo/node_modules/tapable/lib/HookCodeFactory.js:24:12), <anonymous>:6:1)
    at AsyncSeriesHook.lazyCompileHook [as _callAsync] (/Users/fisu/Projects/yoo/node_modules/tapable/lib/Hook.js:35:21)
    at hooks.optimizeTree.callAsync.err (/Users/fisu/Projects/yoo/node_modules/webpack/lib/Compilation.js:939:32)
    at AsyncSeriesHook.eval [as callAsync] (eval at create (/Users/fisu/Projects/yoo/node_modules/tapable/lib/HookCodeFactory.js:24:12), <anonymous>:6:1)
    at AsyncSeriesHook.lazyCompileHook [as _callAsync] (/Users/fisu/Projects/yoo/node_modules/tapable/lib/Hook.js:35:21)
    at Compilation.seal (/Users/fisu/Projects/yoo/node_modules/webpack/lib/Compilation.js:881:27)
    at hooks.make.callAsync.err (/Users/fisu/Projects/yoo/node_modules/webpack/lib/Compiler.js:464:17)
    at _err0 (eval at create (/Users/fisu/Projects/yoo/node_modules/tapable/lib/HookCodeFactory.js:24:12), <anonymous>:11:1)
    at _addModuleChain (/Users/fisu/Projects/yoo/node_modules/webpack/lib/Compilation.js:749:12)
    at processModuleDependencies.err (/Users/fisu/Projects/yoo/node_modules/webpack/lib/Compilation.js:688:9)
    at process._tickCallback (internal/process/next_tick.js:150:11)
error Command failed with exit code 1.
```

**Comment by scinos** [comment]
Being resolved in branch `webpack-4`

**Comment by scinos** [comment]
Should be fixed in v 2.0.0
